bundle/README.md

```diff
@@ -1,3 +1,39 @@
-This bundle contains the operator charms for running MetalLB within a
-Kubernetes cluster. Please see the [documentation](https://ubuntu.com/kubernetes/docs/metallb)
-for more information.
+## Overview
+
+MetalLB offers a software network load balancing implementation that allows for
+LoadBalancing services in Kubernetes. Upstream documentation for MetalLB can be
+found at <https://metallb.universe.tf/>
+
+The official documentation for these charms and how to use them with Kubernetes
+can be found at <https://ubuntu.com/kubernetes/docs/metallb>.
+
+## Deploying
```

Comment:
This `Deploying` section is incomplete/confusing.  You can't deploy ~containers/metallb outside of a k8s cloud, which means we'd have to duplicate https://github.com/charmed-kubernetes/kubernetes-docs/pull/472 here to ensure people know what steps need to happen before this charm is expected to work.

IMHO, the `Overview` is enough.  Leave these deployment / config steps out in favor of the official docs.